2025-T2/T2-HCD/projects/IRL/docs/CHANGELOG.md

```
@@ -0,0 +1,72 @@
+# 3. Set up your initial project structure:
+
+
+cd 2025-T2/T2-HCD/projects/IRL
+mkdir -p src/{core,graphql,db,models,services,middleware,utils}
+mkdir -p dashboard tests k8s docs
+touch package.json tsconfig.json docker-compose.yml .env.example
+```
+
+### **This Week**
+- [X] Complete Phase 0 (Foundation) - 2-3 days
+    - [X] Get a running Node.js server with Redis connected
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)